### PR TITLE
Use semantic markup

### LINKED ISSUE #10: Semantic markup implementation
We need to add semantic markup using schema.org and RDFa in Embedder templates

### PR DESCRIPTION
**Pull request by hvelarde**
closes #10

### CODE REVIEW

File: src/sc/embedder/content/templates/view.pt [hunk 1]
```diff
@@ -7,40 +7,59 @@
...
+
         <div tal:replace="structure provider:plone.abovecontentbody" />
 
-        <div tal:condition='python:context.player_position != "Bottom"'
-            tal:attributes="class view/get_player_pos_class">
-          <span tal:replace="structure context/embed_html"/>
-        </div>
+        <div id="content-core">
+          <metal:content-core define-macro="content-core"
+              tal:define="sameAs context/url|nothing;
```
Comment (rodfersou) [inline src/sc/embedder/content/templates/view.pt]: Can I presume that **context/url** points to the original video url, right?

File: src/sc/embedder/content/templates/view.pt [hunk 2]
```diff
@@ -1,48 +1,65 @@
...
+                          text context/text/output|nothing;
+                          alt_text context/alternate_content/output|nothing">
+            <link property="sameAs" tal:condition="sameAs" tal:attributes="href sameAs">
 
             <div tal:condition='python:context.player_position != "Bottom"'
-                 tal:attributes="class view/get_player_pos_class">
-                 <span tal:replace="structure context/embed_html"/>
+                tal:attributes="class view/get_player_pos_class">
+              <span tal:replace="structure context/embed_html" />
```
Comment (rodfersou) [inline src/sc/embedder/content/templates/view.pt]: how about to add property **video** somehow here

File: src/sc/embedder/content/templates/view.pt [hunk 3]
```diff
@@ -1,48 +1,65 @@
...
+        <div property="description" class="documentDescription"
+            tal:content="context/description">Description</div>
 
-            <div tal:replace="structure provider:plone.belowcontenttitle" />
+        <div tal:replace="structure provider:plone.abovecontentbody" />
 
-            <div tal:replace="structure provider:plone.abovecontentbody" />
+        <div id="content-core">
+          <metal:content-core define-macro="content-core"
+              tal:define="sameAs context/url|nothing;
```
Comment (rodfersou) [inline src/sc/embedder/content/templates/view.pt]: Can I presume that **context/url** points to the original video url, right?

File: src/sc/embedder/content/templates/view.pt (hunk 2 above)
Comment (rodfersou) [inline src/sc/embedder/content/templates/view.pt]: Also there is property **audio** that might be used here..

Comment (hvelarde) [inline src/sc/embedder/content/templates/view.pt]: you don't know if the embedded content is a video.

File: src/sc/embedder/content/templates/view.pt (hunk 3 above)
Comment (hvelarde) [inline src/sc/embedder/content/templates/view.pt]: you can embed anything, not just videos.

**Pull request merged**